addons/mail/static/src/discuss/call/common/call.xml

```diff
@@ -11,6 +11,7 @@
             'position-relative rounded-2 o-mx-0_5 o-mt-0_5 p-1': !state.isFullscreen and !props.isPip,
         }">
             <div class="o-discuss-Call-main d-flex flex-grow-1 flex-column align-items-center justify-content-center position-relative overflow-auto o-scrollbar-thin" t-on-mouseleave="onMouseleaveMain">
+                <BlurPerformanceWarning/>
```

Comment:
By moving logic to JS model, this becomes a `t-if`, which is better over mounting component with conditional on its whole template.

Reason is performance, but also this LOC gives wrong impression a template is necessarily rendered. Besides it incites using flawed `useEffect`.